plugin.yml

```diff
@@ -150,6 +153,7 @@ configuration:
     environment: [ run ]
     expand-volume-vars: [ volumes ]
     graceful-shutdown: [ run ]
+    labels: [ build, run ]
```

Comment:
as far as I understand, this means that the schema will consider the usage of `labels` invalid unless you specify both `build` and `run`, that I don't think it is what you want here